src/StorageApi/StackSpecificComponentIdTranslator.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\ProjectRestore\StorageApi;
+
+use Psr\Log\LoggerInterface;
+
+class StackSpecificComponentIdTranslator
+{
+    private string $destinationStack;
+    private LoggerInterface $logger;
+
+    public function __construct(
+        string $destinationStack,
+        LoggerInterface $logger
+    ) {
+        $this->destinationStack = $destinationStack;
+        $this->logger = $logger;
+    }
+
+    private const STACK_SPECIFIC_COMPONENTS = [
```

Comment:
tady bych doplnil všechny stacky, ta migrační appka se dá použít i na jiné stacky než jen GCP

Reply:
ale při nejhorším to zaloguje jen warning, takže asi v pohodě a nechám to na tobě

Reply:
To jsem na tebe [vykopl tady](https://keboolaglobal.slack.com/archives/C055HSMKX51/p1716901388186219) a Erik se vyjádřil ve smyslu, že ty warningy na non-GCP stacky jsou asi přijatelný zlo. 🙂  Kdyby to moc zasíralo, tak doplním ostatní stacky.